python/examples/9-showing_custom_bonds.py

```diff
@@ -0,0 +1,210 @@
+"""
+Showing custom bonds using stk
+==============================
+
+This example demonstrates how to add shapes into the chemiscope output such
+that custom bonds that would not normally be assigned by ASE can be rendered.
```

Comment:
```suggestion
that custom bonds that would not automatically be assigned can be rendered.
```

the bond assignment is not done by ASE by directly by chemiscope, based on atomic types + distances

Reply:
Done.